pkg/storage/utils/decomposedfs/spaces.go

```diff
@@ -282,7 +282,7 @@ func (fs *Decomposedfs) ListStorageSpaces(ctx context.Context, filter []*provide
 		return nil, errtypes.PermissionDenied(fmt.Sprintf("user %s is not allowed to list spaces of other users", ctxpkg.ContextMustGetUser(ctx).GetId().GetOpaqueId()))
```

Comment:
Don't you want to check for `userId != spaceIDAny` here rather than for `spaceTypeAny`?

Reply:
true, we should probably introduce a new `anyID` const instead of three https://github.com/cs3org/reva/pull/3084/files/9e683197d0f3bd17af4941ac2732a6f46ec77f04#diff-d308a0558595437a537e77e29346472e66ab3bc31c213fcd2dd9b5f64c02aa91R247-R249

not part of this pr, though